docs/fix/empty-wheelbarrows.rst

```diff
@@ -0,0 +1,33 @@
+fix/empty-wheelbarrows
+======================
+
+.. dfhack-tool::
+    :summary: Empties stuck items from wheelbarrows
+    :tags: fort bugfix items
+
+Empties all wheelbarrows which contain rocks that have become 'stuck' in them.
+
+This works around the issue encountered with :bug:`6074`, and should be run
+if you notice wheelbarrows lying around with rocks in them that aren't
+being used in a task.
+
+
+Usage
+-----
+
+``fix/empty-wheelbarrows [options]``
+
+-q, --quiet    surpress console output
+-d, --dry-run  dry run, don't commit changes
```

Comment:
these should go in an `Options` section after the `Examples` section. See https://github.com/DFHack/scripts/blob/master/docs/build-now.rst for an example Options section

Reply:
thanks, done